addins/Cake.Unicorn.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.Unicorn
 NuGet: Cake.Unicorn
-Prerelease: false
 Assemblies:
 - "/**/Cake.Unicorn.dll"
-Repository: https://github.com/georgechang/Cake.Unicorn
+Repository: https://www.nuget.org/packages/Cake.Unicorn/
```

Comment:
This nu spec har no urls in it, should we make on up then?
```xml
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2012/06/nuspec.xsd">
  <metadata>
    <id>Cake.Unicorn</id>
    <version>1.0.14</version>
    <authors>Cake.Unicorn</authors>
    <owners>Cake.Unicorn</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <description>Package Description</description>
    <dependencies>
      <group targetFramework=".NETFramework4.6.1">
        <dependency id="Cake.Core" version="0.30.0" exclude="Build,Analyzers" />
        <dependency id="MicroCHAP" version="1.2.2.2" exclude="Build,Analyzers" />
      </group>
    </dependencies>
    <frameworkAssemblies>
      <frameworkAssembly assemblyName="System.Net.Http" targetFramework=".NETFramework4.6.1" />
      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.6.1" />
    </frameworkAssemblies>
  </metadata>
</package>
```

Reply:
The current logic that we agreed to last year was to fallback on the nuget.org URL if the author has omitted the projectURL from their nuspec.

In this specific case, I have [notified the author](https://github.com/georgechang/Cake.Unicorn/issues/5) about the few discrepancies I found in their nuspec.